src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable */
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <tr
+            data-cy="person"
+            key={person.slug}
+            className={person.slug === slug ? 'has-background-warning' : ''}
+          >
+            <td>
+              <PersonLink person={person} />
+            </td>
+
+            <td>{person.sex}</td>
+            <td>{person.born}</td>
+            <td>{person.died}</td>
+            <td>
+              {people.find(
+                personName => person.motherName === personName.name,
+              ) ? (
+                <PersonLink
+                  person={
+                    people.find(
+                      currPerson => currPerson.name === person.motherName,
+                    )!
+                  }
+                />
+              ) : person.motherName ? (
```

Comment:
try to avoid situation when one condition in another because it makes readability worse